lib/hammer_cli_foreman_remote_execution/options/normalizers.rb

```diff
@@ -1,3 +1,5 @@
+require 'parse-cron'
```

Comment:
I guess, I can remove it, not everyone installs it only on their foreman i guess.